src/functionalTest/java/uk/gov/hmcts/sptribs/caseworker/CaseworkerAddNoteFT.java

```diff
@@ -34,4 +38,16 @@ public void shouldUpdateCaseDataWithNotesWhenAboutToSubmitCallbackIsInvoked() th
             .when(IGNORING_EXTRA_FIELDS)
             .isEqualTo(json(expectedResponse(RESPONSE)));
     }
+
+    @Test
+    public void shouldUpdateCaseDataWithMultipleNotesWhenAboutToSubmitCallbackIsInvoked() throws Exception {
+        final Map<String, Object> caseData = caseData(REQUEST_MULTIPLE_NOTES);
+
+        final Response response = triggerCallback(caseData, EventConstants.CASEWORKER_ADD_NOTE, ABOUT_TO_SUBMIT_URL);
```

Comment:
You can import the event name as static var removing the need for EventConstants.